server/src/main/java/org/elasticsearch/index/search/QueryParserHelper.java

```diff
@@ -143,6 +152,7 @@ private QueryParserHelper() {}
             if (fieldType == null) {
                 // Note that we don't ignore unmapped fields.
                 fields.put(fieldName, weight);
+                unmappedFields.add(fieldName);
```

Comment:
We can ignore unmapped fields here and change the comment ? That's the scope of #49002, we want to ignore unmapped fields so we don't need to forward them to the Lucene parsing.

Reply:
Got it. I thought previously they may still be needed down the pipe. I've updated the PR per your suggestion.